tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -529,8 +546,9 @@ def __validate_t5_atd_and_ssi(self):
                     msg=(
                         f"{t5_model_name} People in states must have a valid value for "
                         f"{self.__get_error_context('REC_SSI', schema)}."
-                    )
+                    ),
+                    deprecated=True
```

Comment:
confirmed that the following error messages deprecated ✔️ 
`T5 People in states must have a valid value for Item 19E (Received Disability Benefits: SSI).`